package.json

```diff
@@ -110,6 +111,7 @@
     "terra-grid": "^6.0.0",
     "terra-heading": "^4.0.0",
     "terra-hookshot": "^5.0.0",
+    "terra-html-table": "^1.0.0",
```

Comment:
I'm not sure if anyone is using the issue form, but if we are still marketing it these entries should be added to the component selection.

https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/IssueForm/Packages.json

Reply:
lol, oh yeah, that thing